utils/exportConfluent/src/main/java/io/apicurio/registry/utils/export/Export.java

```diff
@@ -237,7 +237,7 @@ public void exportSubjectVersionWithRefs(ExportContext context, String subject,
         versionEntity.createdBy = "export-confluent-utility";
         versionEntity.createdOn = System.currentTimeMillis();
         versionEntity.description = null;
-        versionEntity.globalId = -1;
+        versionEntity.globalId = metadata.getId();
```

Comment:
In my view, this should not be the case. Note that, if you have producers/consumers using Confluent's Schema Registry and want to move to Apicurio Registry using this tool, the equivalent of their globalId would be the contentId, not our globalId, thus why the globalId is being autogenerated. At the very minimum, this option should be put behind an option.